components/automate-ui/src/app/app.module.ts

```diff
@@ -356,6 +357,7 @@ import { AppConfigService } from 'app/services/app-config/app-config.service';
     OrgRequests,
     PolicyFileRequests,
     ServiceGroupsRequests,
+    UserPreferencesRequests,
```

Comment:
@Venkatesh-rengasamy please follow the Alphabetical order.

Reply:
done